relay-replays/src/lib.rs

```diff
@@ -1,2 +1,17 @@
+//! Session replay protocol and processing for Sentry.
+//!
+//! [Session Replay] provides a video-like reproduction of user interactions on a site or web app.
+//! All user interactions — including page visits, mouse movements, clicks, and scrolls — are
+//! captured, helping developers connect the dots between a known issue and how a user experienced
+//! it in the UI.
+//!
+//! [session replay]: https://docs.sentry.io/product/session-replay/
```

Comment:
nit, as the case is in the linked docs
```suggestion
//! [Session Replay]: https://docs.sentry.io/product/session-replay/
```

Reply:
That's intentional, my markdown utilities do that automatically. Links are case insensitive, which is a useful feature to support linking to the same document if you use the same keyword multiple times with different casing (e.g. at the beginning and in the middle of sentences).

Great catch, though :)